src/Behat/Behat/Output/Node/EventListener/AST/OutlineListener.php

```diff
@@ -138,8 +138,9 @@ private function printExampleHeaderOnBeforeExampleEvent(Formatter $formatter, Ev
 
         $this->exampleSetupPrinter->printSetup($formatter, $event->getSetup());
 
-        $this->example = $event->getScenario();
-        assert($this->example instanceof ExampleNode);
+        $scenario = $event->getScenario();
```

Comment:
We need to assert that the scenario is an ExampleNode before assigning to the property as the property has the `ExampleNode` type